osu.Game/Online/Leaderboards/Leaderboard.cs

```diff
@@ -151,7 +151,7 @@ protected PlaceholderState PlaceholderState
                         break;
 
                     case PlaceholderState.NotLoggedIn:
-                        replacePlaceholder(new LoginPlaceholder());
+                        replacePlaceholder(new LoginPlaceholder(@"Please sign in to view online leaderboards!"));
```

Comment:
This case should probably be removed and inherit `OnlineViewContainer`.

Reply:
Should be kept for a follow-up PR covering usages  imo.

Reply:
In this case, why is this change even here? It seems orthogonal and unrelated.